ui/app/AppLayouts/Browser/FavoriteMenu.qml

```diff
@@ -0,0 +1,53 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.3
+import QtWebEngine 1.9
+import "../../../shared"
+import "../../../shared/status"
+import "../../../imports"
+import "../Chat/ChatColumn/ChatComponents"
+import "../Profile/LeftTab/constants.js" as ProfileConstants
+
+PopupMenu {
+    property var openInNewTab: function () {}
+    property string url
+    property var currentFavorite: getCurrentFavorite(url)
+
+    id: root
+    closePolicy: Popup.CloseOnEscape | Popup.CloseOnPressOutside
+
+    Action {
+        text: qsTr("Open in new Tab")
+        icon.source: "../../img/generate_account.svg"
+        icon.width: 12
```

Comment:
Is it okay to have icons with different width/height in the same popup menu? I thought it would look weird / lose alignment

Reply:
Ah you're right, there is a slight mis-alignement